src/modules/back-to-top/back-to-top.component.html

```diff
@@ -0,0 +1,3 @@
+<div #backToTop class="back-to-top hidden" (click)="scrollToTop()">
```

Comment:
Stache's classnames should always be prefixed with `stache-` (`stache-back-to-top`).